src/sketch.js

```diff
@@ -193,6 +198,12 @@ function setup()
     // set sketch frame rate
     frameRate(FRAME_RATE);
 
+    // setup amplitude helper in smooth mode
+    amplitude = new p5.Amplitude();
+    amplitude.smooth(SMOOTH_SCALE);
+
+    fft = new p5.FFT(0.4, 16);
```

Comment:
Please comment this line.

Reply:
Also, we should stop creating these variables inside function scopes.

It's ok for those created in `preload` to stay there and be fixed later, but this is a case where it's better done already by "the right way" (in the beginning of the file, in the DEFINITIONS section.

Reply:
👍